### ISSUE TITLE
Linux: Missing Wayland support

### ISSUE THREAD

**Issue by michaelof** [a-sit/PDF-Over#66]
PDF-Over 4.4.4 (and earlier)
OpenSUSE 15.4 (LINUX, Wayland)

- no Drag&Drop possible in step 1 (Öffnen/Open)
- no visual signature placement possible in step 2 (Positionierung/Positioning), screen stays grey

Tested with KDE/Plasma: works fine with X-Server, not with Wayland

Discussed per email already: Known issue, no Wayland support available right now, future enhancement.

**Comment by iaik-jheher** [comment]
We currently try to opt into XWayland-based support by specifying `GDK_BACKEND` in the launch script. Is XWayland available on your system?

**Comment by michaelof** [comment]
"Side effect" of my mistake in #64 

Issue has to be narrowed:

- no Drag&Drop possible in step 1 (Öffnen/Open)

is the only thing missing in Wayland, now, with 4.4.4.

With 4.4.4 it's nicely again possible to use the visual signature placement is working again, great!

Will check for XWayland and report back...

**Comment by iaik-jheher** [comment]
Is this the latest 4.4.4 with #68 included? We updated the installer 1-2 days after the initial release.

**Comment by michaelof** [comment]
NOW yes.
Downloaded 4.4.4 again. 
Deinstalled and (re)installed 4.4.4. 
Drag&Drop still not possible.

**Comment by iaik-jheher** [comment]
Will investigate.

**Comment by michaelof** [comment]
SORRY for late status update:

As of PDF-Over v4.4.5, running under Wayland on OpenSuse 15.6, KDE/Plasma 5.27.11 / KDE Frameworks 5.115.0, Qt 5.15.12, BOTH Drag/Drop AND graphical placement are working like a charm.

(For a while now, but hopefully better late than never confirmed :))

**Comment by iaik-jheher** [comment]
Very strange, we did never manage to reproduce it internally. I can only conclude it was a weird WM quirk that ended up being fixed.

**Comment by mkurz** [comment]
Wayland support is still incomplete; running the application with `GDK_BACKEND=x11` is just a workaround to run it under XWayland. It's not the same as running in natively in Wayland.

Running the application with `GDK_BACKEND=wayland` still causes the original issue reported here:

> - no visual signature placement possible in step 2 (Positionierung/Positioning), screen stays grey

I can reproduce that. Implemented a fix that I tested and I can confirm now works natively in Wayland:
- #183
